Gruntfile.js

```diff
@@ -0,0 +1,31 @@
+/*!
+GPII Universal project Gruntfile
+
+Copyright 2014 RTF-US
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+The research leading to these results has received funding from the European Union's
+Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+"use strict";
+module.exports = function (grunt) {
+    grunt.initConfig({
+        lintAll: {
+            sources: {
```

Comment:
Please update lines below to match file/directory structure of this repo. Thanks.

Reply:
these lines have been updated.